src/components/chats/components/CreateNewChat/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Skeleton } from '@/components/ui/skeleton'
+import { useNavigation } from '@/lib/frontend/useNavigation'
+import { FC } from 'react'
+import { CreateChat } from './CreateChat'
+import { CreateStandaloneChat } from './CreateStandaloneChat'
+import { CreateNewChatProps } from './types'
+
+export const CreateNewChat: FC<CreateNewChatProps> = ({
```

Comment:
We are avoiding to use FC. See this:
https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components

and scroll to _Why React.FC is not needed?_

You can do:
```
export const CreateNewChat = ({
  loading,
  ...props
}: CreateNewChatProps) => {
...
```
and let Typescript infer the rest.

Reply:
I like forcing the return type, I saw many posts but I don't think there's a clear benefit. But if we wan to keep the consistency that's cool

Reply:
Okay I see. I wouldn't mind forcing the return type either, but I'd optimize for consistency. As in, if we do it here we do it everywhere; or we don't do it.